### PR TITLE
skiping-link-for-tenant-in-table-link

### PR DESCRIPTION
**Pull request by Ali-sharafi-git**
skipping link for Admin and tenant account types in admin page

**(This is portal business. I think IUI should not be aware of any business. So we need to move this to portal code.)**

### CODE REVIEW

File: components/molecules/table/formatter/link.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ import { StyledLink } from '../../../'
 import styled from 'styled-components'
 
 export const TableLink = (path, key, value, account) => ({ row }) => {
+  let skipLink = false
+  if (account && ['admin', 'tenant'].includes(row.type.toLowerCase())) skipLink = true
```
Comment (iwarner) [inline components/molecules/table/formatter/link.js:20]: @Ali-sharafi-git Agree should be in portal
@Ali-sharafi-git Always put curly brakcets on conditionals

**Comment by iwarner** [comment]
@Ali-sharafi-git Please resolve the comments

File: components/molecules/table/formatter/link.js [hunk 2]
```diff
@@ -15,14 +15,12 @@ import { StyledLink } from '../../../'
 // Style
 import styled from 'styled-components'
 
-export const TableLink = (path, key, value, account) => ({ row }) => {
-  const getPath = () => {
-    let url = path
-
-    if (account) {
```
Comment (iwarner) [inline components/molecules/table/formatter/link.js:22]: @Ali-sharafi-git Have you tested this exclusion with the pages on portal that swap the Customer / Supplier and Admin links, as this i sremoved is there not a lot of refactor on the portal required?

Comment (Ali-sharafi-git) [reply components/molecules/table/formatter/link.js:22]: account parameter is not passed to this component in most places in portal.so path will be used.
I tested accounts, users, customers, suppliers list pages.
Portal code to change account list and user list is in [this ](https://github.com/DryKISS/portal.cleverly.works/pull/268/files)PR

**Pull request merged**